docs/docs/dev/policies/mesh-trace.md

```diff
@@ -0,0 +1,340 @@
+name: default
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+    name: default
+  default:
+    backends:
+      - datadog:
+          url: http://127.0.0.1:8126
+          splitService: true
+    tags:
+      - name: team
+        literal: core
+      - name: env
+        header:
+          name: x-env
+          default: prod
+      - name: version
+        header:
+          name: x-version
+    sampling:
+      overall:
+        value: 80
+      random:
+        value: 60
+      client:
+        value: 40
+```
+:::
+::: tab "Minimal example"
+
+```yaml
+type: MeshTrace
+name: default
+mesh: default
+spec:
+  default:
+    backends:
+      - datadog:
+          url: http://127.0.0.1:8126
+```
+:::
+::::
+
+Apply the configuration with `kumactl apply -f [..]` or with the [HTTP API](../reference/http-api.md).
+
+:::::
+::::::
+
+The `splitService` property determines if Datadog service names should be split based on traffic direction and destination.
+For example, with `splitService: true` and a `backend` service that communicates with a couple of databases,
+you would get service names like `backend_INBOUND`, `backend_OUTBOUND_db1`, and `backend_OUTBOUND_db2` in Datadog.
+By default, this property is set to false.
+
+:::::::
+::::::::
+
+## Configuration options
+
+### Sampling
+
+You can configure sampling settings equivalent to Envoy's:
+- [overall](https://www.envoyproxy.io/docs/envoy/v1.22.5/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html?highlight=overall_sampling#extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-tracing)
+- [random](https://www.envoyproxy.io/docs/envoy/v1.22.5/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html?highlight=random_sampling#extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-tracing)
+- [client](https://www.envoyproxy.io/docs/envoy/v1.22.5/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html?highlight=client_sampling#extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-tracing)
+
+Example:
+
+```yaml
+sampling:
+  overall:
+    value: 80
+  random:
+    value: 60
+  client:
+    value: 40
+```
+
+### Tags
+
+You can add tags to trace metadata by directly supplying the value ("literal") or by taking it from a header ("header").
+
+Example:
+
+```yaml
+tags:
+  - name: team
+    literal: core
+  - name: env
+    header:
+      name: x-env
+      default: prod
+  - name: version
+    header:
+      name: x-version
+```
+
+For `header` - if a value is missing then `default` will be used.
+If `default` is not provided then the tag won't be added.
+
+## Targeting parts of the infrastructure
+
+With a lot of traffic going through the system,
+a valid use case is to target specific set of microservices that are part of a main request path
+(e.g. a process of buying an item in an e-commerce application).
```

Comment:
I think this paragraph is not super clear.

I think this to make sense you really need to add an example.

Reply:
changed to be the mirror of the "tip" in old docs